src/test/java/uk/gov/pay/adminusers/resources/ServiceResourceIT.java

```diff
@@ -120,18 +148,30 @@ public void should_return_users_ordered_by_email() {
         }
 
         @Test
-        public void should_return_404_if_service_does_not_exist() {
+        void should_return_404_if_service_does_not_exist() {
             givenSetup()
                     .when()
                     .accept(JSON)
                     .get("/v1/api/services/999/users")
                     .then()
                     .statusCode(404);
         }
+        
+        @Test
+        void should_return_400_if_role_does_not_exist() {
+            givenSetup()
+                    .when()
+                    .accept(JSON)
+                    .get(format("/v1/api/services/%s/users?role=charlix", service.getExternalId()))
+                    .then()
+                    .statusCode(400)
+                    .body("message", is("query param role must be one of [ADMIN, VIEW_AND_REFUND, VIEW_ONLY, " +
+                            "VIEW_AND_INITIATE_MOTO, VIEW_REFUND_AND_INITIATE_MOTO, SUPER_ADMIN]"));
+        }
```

Comment:
Not sure how to wrangle this into the lower-case versions but I don't think it's worth the effort just now as it's an internal API.